server/session.ts

```diff
@@ -27,6 +27,7 @@ export default function setupSession(app: Express) {
     app.set('trust proxy', 1);
 
     app.use(
+        /\/((?!api\/toggles).)*/,
```

Comment:
Forstår det som at vi her ønsker å matche alt _utenom_ `/api/toggles`. I så fall tror jeg denne blir mer nøyaktig
```suggestion
        /^(?!\/api\/toggles)/,
```

Reply:
Endret til denne.